app/api/ride.py

```diff
@@ -0,0 +1,49 @@
+# app/ride.py
+
+from fastapi import APIRouter, HTTPException
+
+from app.models.Customer import Customer
+from app.models.Driver import Driver
+from app.models.GeoUtility import GeoUtility
+from app.models.RideManager import RideManager
+from app.models.request_models.booking_request import BookingRequest
+
+
+router = APIRouter()
+
+ride_manager = RideManager()
+driver1= Driver("Rohit Sharma", True, (42.3601, -71.0589))
+driver2= Driver("Virat Kohli", True, (43.1939, -71.5724) )
+
+driver3= Driver("Sachin Tendulkar",  True, (34.0549, -118.2426))
+
+ride_manager.driverManager.addDriver(driver1)
+ride_manager.driverManager.addDriver(driver2)
+ride_manager.driverManager.addDriver(driver3)
+
+@router.get("/ride")
+async def initializeRide():
+    """Get all locations"""
+    return {"message": "Welcome to Uber"}
```

Comment:
Whats happening here ? Wanted to know why this end point .

Reply:
I just had it as a testing endpoint